src/CalcViewModel/StandardCalculatorViewModel.cpp

```diff
@@ -1167,6 +1183,7 @@ Array<unsigned char>^ StandardCalculatorViewModel::Serialize()
     DataWriter^ writer = ref new DataWriter();
     writer->WriteUInt32(static_cast<UINT32>(m_CurrentAngleType));
     writer->WriteBoolean(IsFToEChecked);
+    writer->WriteBoolean(IsFToEAuto);
```

Comment:
Even if `Serialize()` is never used and #370 already plans to remove it, it's better to add new field at the end to stay compatible with the previous version of the deserializer